src/richie/apps/courses/models/course.py

```diff
@@ -885,6 +902,16 @@ def safe_title(self):
         except ObjectDoesNotExist:
             return None
 
+    @property
+    def is_searchable(self):
```

Comment:
To be coherent with the `is_listed` field on the `Course` model, we could name this field `is_listed` as well?

Reply:
In reality we don't need that method. The filter is being made directly.
https://github.com/igobranco/richie/blob/igobranco/issues/1502/course_run_catalog_visibility/src/richie/apps/search/indexers/courses.py#L580

Reply:
ok let's remove it?

Reply:
Done.